src/ImageIO/CGImageAnimation.cs

```diff
@@ -17,7 +17,7 @@
 namespace ImageIO
 {
 
-    public partial class CGImageAnimation
+    public static partial class CGImageAnimation // should remove partial? or leave in case we need to add more to this class in other files later?
```

Comment:
not a big deal... but whenever I see `partial` I must grep to see what else could be present
so **not** having it _by default_ can be helpful